editor/test/integration/script_properties_test.clj

```diff
@@ -296,51 +296,6 @@
         (is (set/superset? (built-resources sub-sub-props-collection)
                            #{(build-resource "/script/resources/from_sub_sub_props_collection.material")}))))))
 
-(defn- created-node [select-fn-call-logger]
```

Comment:
All of this was moved to the `test-util` module and is reused in the non-editable resource tests now.